<ui version="4.0" >
 <class>PyToner</class>
 <widget class="QWidget" name="PyToner" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>PyToner</string>
  </property>
  <widget class="QComboBox" name="brand_combo" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>10</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="brand_label" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Brand</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="product_table" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>40</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="brand_add" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>10</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QToolButton" name="brand_del" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>10</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Del</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
